docs/mkdocs.yml

```diff
@@ -165,6 +168,8 @@ markdown_extensions:
   - pymdownx.emoji:
       emoji_index: !!python/name:material.extensions.emoji.twemoji
       emoji_generator: !!python/name:material.extensions.emoji.to_svg
+  - toc:
+      permalink: "#"
```

Comment:
Also added permalinks that show up when you hover. unrelated to executable changes